### ISSUE TITLE
Sidebar menu tree dropdown always open

### ISSUE THREAD

**Issue by EmanueleCoppola** [ColorlibHQ/AdminLTE#1854]
### Issue type:
- [ ] Feature request 
- [x] Bug report       
- [ ] Documentation   

### Environment:
* AdminLTE Version:  v2.4.0
* Operating System:  Windows 10
* Browser (Version):  Chrome ( 65.0.3325.181 - Latest), Firefox (59.0.2 - Latest)

### Description:
You can reproduce the bug here https://adminlte.io/themes/AdminLTE/pages/examples/profile.html with one of the browser versions that I use, the sidebar dropdown will only stay opened as you can see in this video https://i.gyazo.com/9858b6a02c35372275a1da6478d3aeba.mp4

**Comment by bmx22c** [comment]
Hi,
I figured out how to fix it... kinda.
It works, but it might not be the best thing to do.
Anyway, here is how I fixed it:
In adminlte.js, add
`$(tree).css('cssText', 'display: none !important;');`
after line 1081
and
` else {
        $(data['element']).children('.treeview:not(.menu-open)').children('.treeview-menu').css('cssText','display: none !important');
      }`
at line 1106.
See [this pastebin](https://pastebin.com/2xQ6m6FP) to see what it should look like.
Hope it will help some people.
